### PR TITLE
Upgrading to ASP.NET 1.1.0

### PR DESCRIPTION
**Pull request by sebastienros**

### CODE REVIEW

**Comment by sebastienros** [comment]
Please don't merge, I still need to point to nuget.org.

File: src/Orchard.Cms.Web/Modules/Orchard.Diagnostics/project.json [hunk 1]
```diff
@@ -2,11 +2,11 @@
   "version": "2.0.0-*",
 
   "dependencies": {
-    "Microsoft.Extensions.DependencyInjection.Abstractions": "1.0.0",
-    "NETStandard.Library": "1.6.0",
+    "Microsoft.Extensions.DependencyInjection.Abstractions": "1.1.0",
+    "NETStandard.Library": "1.6.1",
```
Comment (Jetski5822) [inline src/Orchard.Cms.Web/Modules/Orchard.Diagnostics/project.json]: Net standard shouldn't be here

File: src/Orchard.Cms.Web/Modules/Orchard.Identity/project.json [hunk 2]
```diff
@@ -2,8 +2,8 @@
   "version": "2.0.0-*",
 
   "dependencies": {
-    "NETStandard.Library": "1.6.0",
-    "Microsoft.Extensions.DependencyInjection.Abstractions": "1.0.0",
+    "NETStandard.Library": "1.6.1",
```
Comment (Jetski5822) [inline src/Orchard.Cms.Web/Modules/Orchard.Identity/project.json]: No netstandard here

File: src/Orchard.Data.Abstractions/project.json [hunk 3]
```diff
@@ -2,8 +2,8 @@
   "version": "2.0.0-*",
 
   "dependencies": {
-    "Microsoft.Extensions.DependencyInjection.Abstractions": "1.0.0",
-    "NETStandard.Library": "1.6.0",
+    "NETStandard.Library": "1.6.1",
```
Comment (Jetski5822) [inline src/Orchard.Data.Abstractions/project.json]: No netstandard

File: src/Orchard.Deployment.Core/project.json [hunk 4]
```diff
@@ -2,8 +2,8 @@
   "version": "2.0.0-*",
 
   "dependencies": {
-    "Microsoft.Extensions.DependencyInjection.Abstractions": "1.0.0",
-    "NETStandard.Library": "1.6.0",
+    "NETStandard.Library": "1.6.1",
```
Comment (Jetski5822) [inline src/Orchard.Deployment.Core/project.json]: No netstandard

File: src/Orchard.DisplayManagement/project.json [hunk 5]
```diff
@@ -1,15 +1,15 @@
...
+    "Microsoft.Extensions.Logging.Abstractions": "1.1.0",
     "Microsoft.AspNetCore.Mvc.Modules.Abstractions": "1.0.0-*",
     "Orchard.DependencyInjection": "2.0.0-*",
     "Orchard.Environment.Cache.Abstractions": "2.0.0-*",
     "Orchard.Environment.Extensions.Abstractions": "2.0.0-*",
     "Orchard.Environment.Shell.Abstractions": "2.0.0-*",
-    "System.Reflection.Extensions": "4.0.1-*",
+    "System.Reflection.Extensions": "4.3.0",
     "System.Diagnostics.TraceSource": "4.0.0-*",
     "Castle.Core": "4.0.0-alpha001"
```
Comment (Jetski5822) [inline src/Orchard.DisplayManagement/project.json]: Update to beta002

File: src/Orchard.Environment.Shell.Data/project.json [hunk 6]
```diff
@@ -2,7 +2,7 @@
   "version": "2.0.0-*",
 
   "dependencies": {
-    "NETStandard.Library": "1.6.0",
+    "NETStandard.Library": "1.6.1",
```
Comment (Jetski5822) [inline src/Orchard.Environment.Shell.Data/project.json]: Remove

**Pull request merged**